src/NuGet.Services.EndToEnd/Support/PushedPackagesFixture.cs

```diff
@@ -281,7 +281,7 @@ private async Task<List<PackageToPrepare>> InitializePackagesAsync(PackageType p
                 case PackageType.SemVer2DueToSemVer2Dep:
                     packageToPrepare = new PackageToPrepare(Package.Create(new PackageCreationContext
                     {
-                        Id = id,
+                        Id = unicodeId ? id + "пакет包elsökning" : id,

```

Comment:
This is the wrong approach. The `PackageType.<Type>` is reused for other tests. We should introduce a new package type and test on that, instead of modifying the existing package type.

Reply:
Sure, added new type.